sound/hybrid_font.h

```diff
@@ -173,7 +173,11 @@ class HybridFont : public SaberBase {
 	  drag.files_found()) {
 	PlayMonophonic(&drag, &drag);
       } else if (lockup.files_found()) {
-	PlayMonophonic(&lockup, &lockup);
+        if (bgnlock.files_found()) {
+          PlayMonophonic(&bgnlock, &lockup);
+        } else {
+	  PlayMonophonic(&lockup, &lockup);
```

Comment:
indentation is off

Reply:
Not sure how that happened but oops! Lemme fix.